README.md

```diff
@@ -5,7 +5,8 @@
 [![codecov](https://codecov.io/gh/mozilla/grcov/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/grcov)
 ![crates.io](https://img.shields.io/crates/v/grcov.svg)
 
-grcov collects and aggregates code coverage information for multiple source files.
+grcov collects and aggregates code coverage information for multiple source files. 
+It should be possible to run grcov on Linux, OSX and Windows.
```

Comment:
```suggestion
Linux, macOS and Windows are supported.
```

Reply:
Thanks - have updated to be more assertive.